src/app/common/__snapshots__/PaginationWidget.spec.ts.snap

```diff
@@ -45,7 +45,36 @@ exports[`PaginationWidget.vue renders pagination 1`] = `
     <!---->
     
     
-     ‹ Previous 
+    <span
+      class="kong-icon-chevronLeft kong-icon"
+    >
+      <svg
+        fill="none"
+        viewBox="0 0 20 20"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        
+  
+        <title>
+          Back
+        </title>
```

Comment:
`Back Previous`? Does this one and `Next Forward` need hiding? Or is it because the snapshots are reporting the DOM before kongponents has had a chance to remove it?

Reply:
Hm, it’s confusing to me why the `hide-title` prop doesn’t hide it here but in the notification icon. Let me look into that.

Reply:
I can only imagine this to be an issue with the KIcon implementation. It does some DOM manipulation and maybe it doesn’t always remove the `title` correctly.

I added `aria-hidden="true"` to those icons now to prevent those from being announced in screen readers. I don’t think we can fix this on our side.

Reply:
LGTM!

What's the deal with kongponents, is it worth filing an issue over there or do you think it is just a test/snapshotted too early thing?

Reply:
Not sure. I don’t exactly know where it breaks and it’s definitely not a pressing matter. If I have some extra time, I can look into it more, but that’s not today.